admin/src/main/resources/io/buoyant/admin/js/client_success_graph.js

```diff
@@ -0,0 +1,69 @@
+var ClientSuccessRateGraph = (function() {
+  var neutralLineColor = "#878787"; // greys.neutral
+
+  function createChartLegend(successLineColor) {
+    return {
+      topLine: neutralLineColor,
+      successRate: successLineColor
+    }
+  }
+
+  function initializeChart($chartEl, dataKeys, timeseriesParamsFn) {
+    var $canvas = $("<canvas id='canvas' height='141'></canvas>");
```

Comment:
lol we should not have so many charts all with id of "canvas" (id is supposed to guarantee uniqueness)